www/src/templates/component-page-template.jsx

```diff
@@ -35,17 +42,28 @@ export default function PageTemplate({
       h6: (props) => <LinkedHeading h="6" {...props} />,
       pre: props => <div {...props} />,
       code: CodeBlock,
+      guide: (props) => (
```

Comment:
Might be kind of a pain to change everywhere, but I'm thinking `testguide` might be a bit clearer to make it more explicit it has to do with the testing guidelines? What do you think?